rca/project_styleguide/templates/patterns/pages/events/event_listing.html

```diff
@@ -0,0 +1,143 @@
+                        <div class="filter-tab-options__footer layout__start-one layout__span-two layout__@large-start-two layout__@large-span-four grid">
+                            <div class="filter-tab-options__clear layout__start-one layout__span-one layout__@large-start-two">
+                                <a class="filter-tab-options__clear-link hidden" href="#" data-filters-clear-category>Clear</a>
+                            </div>
+                            <button class="filter-tab-options__button link link--primary layout__start-two layout__span-one layout__@large-start-four" data-filter-submit>
+                                <span class="link__label">Show results</span>
+                                <svg width="12" height="8" class="link__icon link__icon--rotated" aria-hidden="true"><use xlink:href="#arrow"></use></svg>
+                            </button>
+                        </div>
+                    </div>
+
+                </div>
+            </section>
+
+            <section class="filter-bar filter-bar--small bg bg--light" data-filter-bar-small>
+                <a class="filter-bar__link body body--two" href="#filters-active" data-filter-launcher>
+                    <span class="filter-bar__label">Filters</span>
+                    <svg width="12" height="8" class="filter-bar__icon" aria-hidden="true">
+                        <use xlink:href="#arrow"></use>
+                    </svg>
+                </a>
+            </section>
+
+            <div class="filter-takeover bg bg--light">
+                <div class="filter-takeover__container">
+                    {% for item in filters.items %}
+                        {% if item.queryset %}
+                            <div class="filter-takeover__tab-content tabs__panel tabs__panel--hidden js-tab-panel" id="{{ item.tab_title|slugify }}" role="tabpanel" aria-labelledby="tab-{{ item.tab_title|slugify }}">
+                                {% include "patterns/molecules/filter-tab-options/filter-tab-options.html" %}
+                            </div>
+                        {% endif %}
+                    {% endfor %}
+                </div>
+            </div>
+
+
+            <section class="section bg bg--light" >
+
+                {% if featured_editorial and show_picks %}
+                    <div class="section__row section__row--first-small section__row--last-extra-small grid">
+                        <h2 class="heading heading--four layout__start-one layout__span-two layout__@large-start-two">
+                            Editor's pick
+                        </h2>
+                    </div>
+
+                    <div class="section__content grid">
+                        {% include "patterns/organisms/carousel/carousel.html" with carousel=featured_editorial modifier='carousel carousel--no-margin' datatag='data-carousel' %}
+                    </div>
+                {% endif %}
+
+                {% if results %}
```

Comment:
should this be `result_count` or is it populated by some other method?

Reply:
Ah the results appear underneath and you show the count if there are results... makes sense : )